File: oscexchange/src/main/java/ac/at/fhstp/digitech/oscexchange/errors/OSCPortOpeningError.java
package ac.at.fhstp.digitech.oscexchange.errors;

import ac.at.fhstp.digitech.oscexchange.OSCPort;

public final class OSCPortOpeningError extends OSCPortError {

    public OSCPortOpeningError(Exception exception, OSCPort port) {
        super(exception, port);
    }

}

File: oscexchange/src/main/java/ac/at/fhstp/digitech/oscexchange/errors/OSCPortOpeningError.kt
package ac.at.fhstp.digitech.oscexchange.errors

import ac.at.fhstp.digitech.oscexchange.OSCPort

/**
 * An error that occurs when a closed port cannot be opened
 */
class OSCPortOpeningError(exception: Exception?, port: OSCPort) :
    OSCPortError(exception, port)